modules/virt-deleting-virt-cli.adoc

```diff
@@ -4,45 +4,39 @@
 
 :_content-type: PROCEDURE
 [id="virt-deleting-virt-cli_{context}"]
-= Deleting {VirtProductName}
+= Uninstalling {VirtProductName} by using the CLI
 
-You can delete {VirtProductName} by using the CLI.
+You uninstall {VirtProductName} by deleting the `HyperConverged` custom resource (CR), the {VirtProductName} Operator subscription, and the `ClusterServiceVersion` (CSV) of the Operator from the `openshift-cnv` namespace.
 
 .Prerequisites
 
-- Install the OpenShift CLI (`oc`).
-
-- Access to a {VirtProductName} cluster using an account with
-`cluster-admin` permissions.
-
-[NOTE]
-====
-When you delete the subscription of the {VirtProductName} operator in the OLM by using the CLI, the `ClusterServiceVersion` (CSV) object is not deleted from the cluster. To completely uninstall {VirtProductName}, you must explicitly delete the CSV.
-====
+- Access to an {product-title} cluster using an account with `cluster-admin` permissions.
+- `oc` command installed on workstation.
 
 .Procedure
 
-. Delete the `HyperConverged` custom resource:
+. Delete the `HyperConverged` CR:
 +
 [source,terminal]
 ----
 $ oc delete HyperConverged kubevirt-hyperconverged -n openshift-cnv
 ----
 
-. Delete the subscription of the {VirtProductName} operator in the Operator Lifecycle Manager (OLM):
+. Delete the {VirtProductName} Operator subscription:
 +
 [source,terminal]
 ----
 $ oc delete subscription kubevirt-hyperconverged -n openshift-cnv
 ----
 
-. Set the cluster service version (CSV) name for {VirtProductName} as an environment variable:
+. Set the {VirtProductName} Operator CSV environment variable:
```

Comment:
I see this is existing text, but it seems a bit much to have folks set a env var for an operator they are going to remove. Maybe just get the CSV name and `oc delete csv {csv_name} -n openshift-cnv`. But that's likely just a me thing.

Reply:
They need the name and current version, not just the CSV name. This would be a better way to do it:

. Obtain the current CSV for the {VirtProductName} Operator:
$ oc get subscription kubevirt-hyperconverged -n openshift-cnv -o yaml | grep currentCSV

Reply:
you can simply identify it by label:
```
$ oc delete csv -n openshift-cnv -l operators.coreos.com/kubevirt-hyperconverged.openshift-cnv
```